server/tidb_test.go

```diff
@@ -215,6 +217,56 @@ func (ts *tidbTestSuite) TestStatusAPIWithTLS(c *C) {
 	server.Close()
 }
 
+func (ts *tidbTestSuite) TestStatusAPIWithTLSCNCheck(c *C) {
+	c.Skip("need add ca-tidb-test-1.crt to OS")
+	root := filepath.Join(os.Getenv("GOPATH"), "/src/github.com/pingcap/tidb")
+	ca := filepath.Join(root, "/tests/cncheckcert/ca-tidb-test-1.crt")
```

Comment:
This test really could not pass on windows :-)

Reply:
yes, this case also need install ca to operation system for linuxer, so `c.Skip` at this time :disappointed: